controllers/drplacementcontrol_controller_test.go

```diff
@@ -1467,12 +1467,16 @@ func runFailoverAction(placementObj client.Object, fromCluster, toCluster string
 	//       resource is cleaned up or not, number of MW may change.
 	if !isSyncDR {
 		Expect(getManifestWorkCount(toCluster)).Should(BeElementOf(3, 4)) // MW for VRG+DRCluster+NS
-		Expect(getManifestWorkCount(fromCluster)).Should(Equal(2))        // DRCluster + NS MW
```

Comment:
Why the test changed for async? fencing is done only for sync.

Reply:
@nirs , the code was changed for async, because I moved the line Expect(getManifestWorkCount(fromCluster)).Should(Equal(2))
outside - it is common for both sync and async.